libs/shared/src/consts/providers/provider.interface.ts

```diff
@@ -27,6 +27,16 @@ export const mailJsConfig: IConfigCredentials[] = [
     displayName: 'Secret key',
     type: 'string',
   },
+  {
+    key: 'fromSender',
+    displayName: 'From sender',
+    type: 'string',
+  },
```

Comment:
I wonder, maybe worth just adding this statically to email providers? I'm afraid we might mispell the naming in some provider and we really don't have a TS validation on it here. We can either do it in the modal itself just checking if the integration is Email, add it automatically those two fields.

What do you think? I do love this approach as it works on the same mechanics as the other code, but I'm lacking some safety net in case we mispell this non-provider specific prop.

Reply:
I am not sure the exact issue you referring to, if the credentials are included in the list they will be reflected in the modal. I don't think that the modal needs extra logic on the client side. 

However, I thought about the duplications on the provider and started to implement that with a help of classes and inheritance however that refactor was not valid because we use both SMS provider and E-Mail provider under the same 'provider list'.

Now that I am thinking of it I can add extra login in order to reuse the credentials, let me show it to you on the next commit and please tell me what you think regarding it.

FYI there were some typos and duplications that I will fix on the next commit.